internal/pkg/agent/application/coordinator/coordinator_test.go

```diff
@@ -447,7 +447,7 @@ func TestCoordinator_Upgrade(t *testing.T) {
 	require.NoError(t, err)
 	cfgMgr.Config(ctx, cfg)
 
-	err = coord.Upgrade(ctx, "9.0.0", "", nil, true)
```

Comment:
Nice little test, it will be fun to see what happens when we will release 9.0 xD